test/new-e2e/pkg/runner/ci_profile.go

```diff
@@ -15,9 +15,13 @@ import (
 
 const (
 	defaultCISecretPrefix = "ci.datadog-agent."
-	defaultCIEnvironments = "aws/agent-qa"
 )
 
+var defaultCIEnvironments = map[string]string{
+	"aws": "agent-sandbox",
+	"az":  "agent-sandbox",
+}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should `defaultCIEnvironemnts` be `agent-qa` instead?
```suggestion
var defaultCIEnvironments = map[string]string{
	"aws": "agent-qa",
	"az":  "agent-qa",
}
```